modules/articles/client/views/view-article.client.view.html

```diff
@@ -4,9 +4,9 @@ <h1 ng-bind="article.title"></h1>
   </div>
   <div class="pull-right" ng-show="authentication.user._id == article.user._id">
     <a class="btn btn-primary" ui-sref="articles.edit({articleId: article._id})">
-      <i class="glyphicon glyphicon-edit"></i>
+      <i class="glyphicon glyphicon-edit"></i> edit
     </a>
```

Comment:
This is not how Materialize does icons - it's how Bootstrap does. Since we might yank bootstrap later, we should probably use materialize icons:
http://materializecss.com/icons.html